### ISSUE TITLE
Graphs have parallel edges

### ISSUE THREAD

**Issue by iamgroot42** [harvardnlp/botnet-detection#11]
It seems the data pre-processing does not convert the graphs to simple graphs. Even though they are undirected and unweighted, some nodes in the graphs have multiple parallel edges. This bug seems to impact graphs loaded via `torch_geometric ` and `dgl`  - `networkx` already handles this.

Thankfully, the number of such parallel edges is not significant - I did a quick check on the train and validation set, and the graphs have ~ 11 extra parallel edges on average. Not a big issue (it should not impact model performance or results in any way), though.

**Comment by jzhou316** [comment]
Hi @iamgroot42 thanks for pointing this out! We saw your pull request and will merge it. Otherwise, `torch_geometric` also has a data [method](https://github.com/rusty1s/pytorch_geometric/blob/4e47b6fc72da673fa77d4c7017aeed92f41a974f/torch_geometric/data/data.py#L271) to check or remove multi-edges, and multi-edges could also be useful in other contexts (e.g. when there are different edge labels, or when the number of multi-edges represents the frequency or importance of that edge when we superpose edges at different times into a single graph).
